starter-code/css/main.css

```diff
@@ -6,3 +6,117 @@ Green: #00B172
 White: #FFF
 
 */
+
+body{
+  font-family: sans-serif;
+  margin: 0px;
+  padding: 0px;
+}
+
+header{
+  height: 80px;
+  margin: 0px;
+}
+
+h1{
+  font-size: 80px;
+  font-weight: normal;
+  color: #FFF;
+}
+
+h2{
+  font-size: 28px;
+  font-weight: lighter;
+  color: #FFF;
+}
+
+.logo{
+  display: inline-block;
+  position: absolute;
+  height: 100%;
+  width: 200px;
+  background-image: url(../images/spotify-logo.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  margin: 10px;
+  animation-name: shake;
+  animation-duration: 0.8s;
+  animation-iteration-count: 1;
+  animation-timing-function:ease-in-out;
+  animation-play-state:paused;
+}
+
+.logo:hover {
+  animation-play-state:running;
```

Comment:
To make the div animate on hover you need to give the animation the name you use when you call it in the keyframes. I leave you here an example:

https://stackoverflow.com/questions/28345222/css3-keyframe-animation-which-runs-on-hover